src/frontend/Sider/Behandling/Inngangsvilkår/Vilkårperioder/EndreFellesFelter.tsx

```diff
@@ -3,26 +3,31 @@ import React from 'react';
 import { FormErrors } from '../../../../hooks/felles/useFormState';
 import { useTriggRerendringAvDateInput } from '../../../../hooks/useTriggRerendringAvDateInput';
 import DateInputMedLeservisning from '../../../../komponenter/Skjema/DateInputMedLeservisning';
-import SelectMedOptions from '../../../../komponenter/Skjema/SelectMedOptions';
+import SelectMedOptions, { SelectOption } from '../../../../komponenter/Skjema/SelectMedOptions';
 import { FeilmeldingMaksBredde } from '../../../../komponenter/Visningskomponenter/FeilmeldingFastBredde';
 import { Periode } from '../../../../utils/periode';
-import { AktivitetType, aktivitetTypeOptions } from '../typer/aktivitet';
+import { AktivitetType } from '../typer/aktivitet';
+import { MålgruppeType } from '../typer/målgruppe';
 
 export interface FellesFormFelter extends Periode {
-    type: AktivitetType | '';
+    type: AktivitetType | MålgruppeType | '';
 }
 
-export const EndreFellesFelter: React.FC<{
+interface Props {
     form: FellesFormFelter;
-    oppdaterTypeIForm: (type: AktivitetType) => void;
+    oppdaterTypeIForm: (type: string) => void;
```

Comment:
Hadde det gitt mening å gjøre denne komponenten generisk for å unngå typecasting? 
Eks
https://github.com/navikt/tilleggsstonader-sak-frontend/commit/431a817db73d7692b43739cc8bd9582d21dcd929